src/components/MultiHopTrade/hooks/useReceiveAddress.tsx

```diff
@@ -39,21 +38,23 @@ export const useReceiveAddress = ({
 
   // Selectors
   const buyAsset = useAppSelector(selectInputBuyAsset)
-  const sellAssetAccountId = useAppSelector(selectFirstHopSellAccountId)
-
-  const buyAccountId = useAppSelector(state =>
-    selectLastHopBuyAccountId(state, { accountId: sellAssetAccountId }),
-  )
```

Comment:
again, the sell asset account id parameter is handled internally within `selectLastHopBuyAccountId`